### ISSUE TITLE
Appnexus keywords in Prebid Server differ from normal Prebid.js

### ISSUE THREAD

**Issue by dbemiller** [prebid/Prebid.js#2647]
## Type of issue
Bug? At the very least, it's surprising behavior for publishers.

## Description
In [Prebid.js](http://prebid.org/dev-docs/bidders.html#appnexusAst), the appnexus `keyword` bidder params look like:

```
  "keywords": {
    "foo": ["bar", "baz"],
    "fizz": ["buzz"]
  }
}
```

In [Prebid Server](https://github.com/prebid/prebid-server/blob/master/static/bidder-params/appnexus.json#L28), we expect them to look like:

```
{
  "keywords": [
    {
      "key": "foo",
      "value": ["bar", "baz"]
    },
    {
      "key": "fizz",
      "value": ["buzz"]
    },
  ]
}
```

Publishers who follow [the docs](http://prebid.org/dev-docs/get-started-with-prebid-server.html) expect the same params to be used for server- or client- side auctions.

## Feature request

I wish I could fix this on the server... the prebid.js API is better. Unfortunately these params are stored in the database, so I can't do much about it on my side.

Could we hack it into [the other param conversions](https://github.com/prebid/Prebid.js/blob/master/modules/prebidServerBidAdapter.js#L207)?

Relates to #2420

**Comment by josephtyler** [comment]
This bug is what I'm seeing, except the examples are swapped. 

Prebid Server looks like this:

```
{
  "keywords": {
    "foo": ["bar", "baz"],
    "fizz": ["buzz"]
  }
}
```

Prebid.js like this:

```
{
  "keywords": [
    {
      "key": "foo",
      "value": ["bar", "baz"]
    },
    {
      "key": "fizz",
      "value": ["buzz"]
    },
  ]
}
```

**Comment by jsnellbaker** [comment]
@dbemiller To confirm, this restructuring of the keyword data is needed for both pbs endpoints?

I tried testing it and saw the error @josephtyler reported for the openrtb endpoint and the following error for the legacy endpoint:
```
json: cannot unmarshal object into Go struct field appnexusParams.keywords of type []appnexus.KeyVal
```

**Comment by jsnellbaker** [comment]
Should be fixed in next release (1.14.0)
